src/components/CheckboxFacet/CheckboxFacetList.js

```diff
@@ -20,15 +20,22 @@ function CheckboxFacetList({
   onMoreClick,
   onSearch,
   onChange,
+  onFetch = () => null,
```

Comment:
Please add it to propTypes 
The default value could be defined in the default propTypes

Reply:
Thanks, fixed